radio/src/FreeRTOSConfig.h

```diff
@@ -20,15 +20,20 @@
 
 #pragma once
 
+#if defined(PCBSKY9X)
+#include "stdint.h"
+extern uint32_t SystemCoreClock;
+#else
 #include "board_common.h"
+#endif
 
 #define configUSE_PREEMPTION            1
 #define configUSE_IDLE_HOOK             0
 #define configUSE_TICK_HOOK             0
 #define configCPU_CLOCK_HZ              ( SystemCoreClock )
 #define configTICK_RATE_HZ              ( ( TickType_t ) 1000 )
 #define configMAX_PRIORITIES            ( 5 )
-#define configMINIMAL_STACK_SIZE        ( ( unsigned short ) 130 )
+#define configMINIMAL_STACK_SIZE        ( ( unsigned short ) 30 )
```

Comment:
Is that really sufficient for the idle task? That would be cool! Saves another 100 bytes RAM!

Reply:
Tested on x7 and tx16s so far, can't see an issue, I GAIN 8 bytes on x7 compared to Coocox with this